plugins/metadata/lib/kMetadataManager.php

```diff
@@ -806,9 +806,11 @@ public static function getDataElasticSearchValues(Metadata $metadata, array &$me
 			KalturaLog::err('Could not load metadata xml [' . $xmlString . '] - ' . $ex->getMessage());
 			return;
 		}
-
 		$profileFields = MetadataProfileFieldPeer::retrieveActiveByMetadataProfileId($metadata->getMetadataProfileId());
 		$metadataId = $metadata->getId();
+		$partnerId = $metadata->getPartnerId();
+		$partnerMaxMetadataLengthConf = kConf::get(ElasticSearchPlugin::MAX_METADATA_INDEX_LENGTH, ElasticSearchPlugin::ELASTIC_DYNAMIC_MAP, array());
```

Comment:
this can be done in one line - 
 maxMetadataLength  = kConf::getArrayValue(partnerId, ElasticSearchPlugin::MAX_METADATA_INDEX_LENGTH, ElasticSearchPlugin::ELASTIC_DYNAMIC_MAP,kElasticSearchManager::METADATA_MAX_LENGTH);